.github/workflows/release.yml

```
@@ -0,0 +1,42 @@
+name: Release
+
+on: [push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Prepare repository
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git checkout ${GITHUB_REF:11} --
+          git remote rm origin
+          git remote add origin "https://$GH_TOKEN@github.com/ericclemmons/codelift.git"
```

I think that won't work, because `GITHUB_TOKEN` is an installation token. If you run into authentication problems with git operations, try

```suggestion
          git remote add origin "https://x-access-token:$GH_TOKEN@github.com/ericclemmons/codelift.git"`
```

See https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#http-based-git-access-by-an-installation

---

Thanks!  Wait, are these alerts spamming you??

---

Not at all, happy to see people using my work :) I can always unsubscribe